lms/static/js/certificates/views/certificate_invalidation_view.js

```diff
@@ -83,16 +84,20 @@
                     if (model) {
                         model.destroy({
                             success: function() {
-                                self.showMessage(gettext('The certificate for this learner has been re-validated and ' +
-                                    'the system is re-running the grade for this learner.'));
+                                self.escapeAndShowMessage(
+                                    gettext('The certificate for this learner has been re-validated and ' +
```

Comment:
I found this in the [i18n docs](http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html):

> However, in Javascript and other languages, the gettext call cannot be broken up over multiple lines. You will have to live with long lines on gettext calls, and we make a style exception for this.